lib/Updater.php

```diff
@@ -137,6 +137,11 @@ public function checkForUpdate() {
 		$versionString = isset($response['versionstring']) ? $response['versionstring'] : '';
 
 		if ($version !== '' && $version !== $this->currentVersion) {
+			if (PHP_INT_SIZE < 8 && version_compare($version, '26.0.0.0', '>=')) {
+				$this->updateAvailable = false;
+				$updateText = '<br />You are running a 32-bit system. Nextcloud 26 supports 64-bit only, therefore an update cannot be offered. Please switch to a 64-bit system first.';
```

Comment:
Is that not translated? Is that expected?

Reply:
see my answer below.